### DISCUSSION TITLE
How to speed up webcam inference ?

### DISCUSSION THREAD

**Discussion by Samuel-Bachorik** [ultralytics/yolov5#2504] [General]
Hi i just want to ask if is any way to speed up real time inference from webcam. My hardware is good, RTX 2080 TI.... And i get really poor results. Few FPS maybe about 3. Simply it is really very laggy. Iam running from detect.py. Any ways ? 

Thank you

**Comment by glenn-jocher** [comment]
@Samuel-Bachorik results will vary by hardware. GPU is important but so is everything else, i.e. CPU and RAM, and if loading files then your device read speeds.
